CONTRIBUTING.md

```diff
@@ -7,7 +7,10 @@
 
 # Prerequisites:
 
-Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+- Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+- Install the **ASP.NET and web development** and **.NET desktop development** workloads in VisualStudio
+- Install the latest version of **.NET Core SDK** for your specific platform. (https://dotnet.microsoft.com/download)
```

Comment:
Would it be sufficient to install the ".NET Core cross-platform development" workload?

Reply:
You are right, .NET Core cross-platform development is sufficient. Also I couldn't use .vsconfig because that will mean updating all the solution directories. Or having the user import it, which doesn't offer any advantage over just installing the workload.